yt_dlp/utils.py

```diff
@@ -5658,7 +5658,11 @@ def windows_enable_vt_mode():
     ENABLE_VIRTUAL_TERMINAL_PROCESSING = 0x0004
 
     dll = ctypes.WinDLL('kernel32', use_last_error=False)
-    handle = os.open('CONOUT$', os.O_RDWR)
+
+    try:
+        handle = os.open('CONOUT$', os.O_RDWR)
+    except OSError: # GUI environment
+        return
```

Comment:
```suggestion
    # `CONOUT$` might not always exist in GUI environments
    # Ref: https://github.com/KurtBestor/Hitomi-Downloader/issues/5592
    try:
        handle = os.open('CONOUT$', os.O_RDWR)
    except OSError:
        return
```
This also fixes the linter complaint